core/templates/dev/head/editor/feedback_tab/ThreadDataService.js

```diff
@@ -81,14 +87,15 @@ oppia.factory('threadDataService', [
         }
       });
     },
-    addNewMessage: function(threadId, newMessage, newStatus, successCallback, errorCallback) {
+    addNewMessage: function(
+      threadId, newMessage, newStatus, successCallback, errorCallback) {
```

Comment:
Optional: we tend to indent lines like this an additional 2 spaces to distinguish from the next line. Ditto for line 129.

(It's OK either way since the linter doesn't catch it, but I thought I'd mention it since you might find it easier to read, too.)